scripts/ci/crates.py

```diff
@@ -331,55 +366,90 @@ def is_already_uploaded(version: str, crate: Crate) -> bool:
     return False
 
 
+def get_retry_delay(error_message: str) -> float | None:
```

Comment:
Please add a comment with an example input error message (or at least the relevant part of it)

Reply:
Also, what unit is this returning? `get_retry_delay_secs` would be more descriptive